nautobot/core/tests/test_api.py

```diff
@@ -488,3 +488,33 @@ def test_related_by_invalid(self):
             response = self.client.post(url, data, format="json", **self.header)
         self.assertHttpStatus(response, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(ipam_models.VLAN.objects.filter(name="Test VLAN 100").count(), 0)
+
+
+class APIOrderingTestCase(testing.APITestCase):
+    """
+    Testing integration with DRF's OrderingFilter.
```

Comment:
Is this test redundant with the generic tests in the above file now? Certainly seems very similar.

Reply:
Yeah could probably safely delete it. I _might_ want to use it though to ensure coverage on different field types (datetime, text, int, etc) instead of exhaustively testing different models to ensure good support.